src/main/java/com/algorand/algosdk/crypto/Verifier.java

```diff
@@ -0,0 +1,47 @@
+package com.algorand.algosdk.crypto;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+import java.io.Serializable;
+
+/**
+ * A serializable representation of a state proof key.
+ */
+@JsonInclude(JsonInclude.Include.NON_DEFAULT)
+public class Verifier implements Serializable {
```

Comment:
The name `Verifier` works in go-algorand because it's scoped in a package, so every reference to it must be `merklekeystore.Verifier`. For the SDKs, I think we'll want a more descriptive name. Maybe `MerkleVerifier` or `StateProofVerifier`?